apps/documentation/src/routes/documentation/contributing.mdx

```diff
@@ -14,35 +14,35 @@
 
 ## TL;DR
 
-The project is massive - if you like it do consider to contribute!
+The project is massive - if you like it, do consider contributing!
 
 ## Getting started
 
-The `tuono` project can mostly be split by the following sub-domains:
+The `tuono` project can mostly be split into the following subdomains:
 
 - The CLI
-- The rust backend
-- The react frontend
+- The Rust backend
+- The React frontend
 - The documentation website (which is written with tuono 🚀)
 
-To check what are the knowledge requirement for each domain check the
+To check what the knowledge requirements are for each domain, check the
 [requirements](#requirements) section below.
 
-Currently I'm keeping a private dashboard to prioritize new features and bug fixes but if you
-want to propose something please open a new issue on github or reach me using
+Currently, I'm keeping a private dashboard to prioritize new features and bug fixes, but if you
+want to propose something, please open a new issue on Github or reach out to me using
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The official name of this software platform is spelled with a capital “H”. (GITHUB[1])
Suggestions: `GitHub`
URL: https://en.wikipedia.org/wiki/GitHub 
Rule: https://community.languagetool.org/rule/show/GITHUB?lang=en-US&subId=1
Category: CASING
<!-- __reviewdog__:ChBmMTg1ZDY2MTRiYjJjMTY4EgxMYW5ndWFnZVRvb2w= -->